### ISSUE TITLE
ffmpeg missing dependency

### ISSUE THREAD

**Issue by cstrouse** [skycocker/chromebrew#1224]
After installing the ffmpeg package it fails to run on XE500C13-K01US (stable channel, X86_64) with the following error:
```
chronos@localhost ~ $ ffmpeg 
ffmpeg: error while loading shared libraries: libxcb.so.1: cannot open shared object file: No such file or directory
```

**Comment by jam7** [comment]
Sorry for inconvinience.  I compiled ffpmeg for x86_64 on the machine xcb and other X11 stuff is installed unfortunately.  I'll recompile it.  In the mean time, please use `crew install ffmpeg -s`.

**Comment by cstrouse** [comment]
Fixed by #1235.
